packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -803,7 +812,7 @@ export default function makeKernelKeeper(
     assert.typeof(name, 'string');
     const k = `vat.name.${name}`;
     assert(kvStore.has(k), X`vat name ${name} must exist, but doesn't`);
-    return kvStore.get(k);
+    return kvStore.get(k) || assert.fail('.has() ensures .get()');
```

Comment:
ok yeah I guess I wasn't being consistent here.. this has+get pair could be replaced by a single `getRequired`

(I think we didn't have `getRequired` when that was first written)

Reply:
I think I tried using `getRequired` here but it changes the error message and there's a test for the error message.